src/app/shared/components/breadcrumb.tsx

```diff
@@ -83,7 +83,7 @@ export default class BreadcrumbComponent extends React.Component<BreadCrumbProps
                 const deviceId = this.getDeviceIdFromSearch(search);
                 const interfaceId = this.getInterfaceIdFromSearch(search);
                 items.push({
-                    href: `#/${ROUTE_PARTS.DEVICES}/${ROUTE_PARTS.DETAIL}/${ROUTE_PARTS.DIGITAL_TWINS}/${ROUTE_PARTS.INTERFACES}/?${ROUTE_PARAMS.DEVICE_ID}=${deviceId}&${ROUTE_PARAMS.INTERFACE_ID}=${interfaceId}`,
+                    href: `#/${ROUTE_PARTS.RESOURCE}/${hubName}/${ROUTE_PARTS.DEVICES}/${ROUTE_PARTS.DEVICE_DETAIL}/${ROUTE_PARTS.DIGITAL_TWINS}/${ROUTE_PARTS.INTERFACES}/?${ROUTE_PARAMS.DEVICE_ID}=${deviceId}&${ROUTE_PARAMS.INTERFACE_ID}=${interfaceId}`,
```

Comment:
actually wondering if module identity / module twin should be children of device.  this pathing would make sense if device was part of route.  But I think we may be walling ourselves into a problem -- forcing us to fetch the entire device and all modules should a user bookmark a module.

Reply:
I am slightly confused by this comment. The api path is `devices/${parameters.deviceId}/modules` for modules, so why would module identity not part of a device?